test/conformance/ptp/ptp.go

```diff
@@ -538,9 +568,14 @@ func configurePTP() {
 	if err == nil && configureFifo {
 		ptpSchedulingPolicy = "SCHED_FIFO"
 	}
-
 	for _, gmInterface := range ptpGrandMasterNode.InterfaceList {
+		if gmInterface != masterInterfaceName && masterInterfaceName != "" {
```

Comment:
There should be a separate validation to either fail if the desired (via env) interface is not in the master interface list/slave interface list  OR  if not found then pick up the default path and pick the default interface the way it did.

Reply:
I did want the changes to be as minimal as possible. Hence, hijacking the current for loop. I'll add separate path and fall back to the regular discovery if it did fail